crossing-training.lic

```diff
@@ -103,6 +104,8 @@ class CrossingTraining
 
     check_tithe
 
+    check_attunement
```

Comment:
Having this always enabled breaks the ability to control attunements priority in between hunts. Crossing training will skip over attunement proper in favor of a lower listed skill because it's at 1/34, so I'd leave town training with 1 mindstate of attunement and 5-6 mind states of other skills I consider less important.